atlasdb-feign/src/main/java/com/palantir/atlasdb/http/AtlasDbFeignTargetFactory.java

```diff
@@ -64,14 +64,16 @@ private AtlasDbFeignTargetFactory() {
 
     public static <T> T createProxy(
             Optional<SSLSocketFactory> sslSocketFactory,
+            Supplier<Optional<String>> authTokenSupplier,
```

Comment:
why does the token need to be live-reloadable, but everything else (uri, ssl) not?